ui/src/main/java/edu/wpi/grip/ui/pipeline/AddSourceView.java

```diff
@@ -48,6 +59,7 @@
     private final MultiImageFileSource.Factory multiImageSourceFactory;
     private final ImageFileSource.Factory imageSourceFactory;
     private final CameraSource.Factory cameraSourceFactory;
+    private final HttpSource.Factory httpSourceFactory;
```

Comment:
This is going to stay a private field to stay consistent with the rest of the source factories.